### PR TITLE
Throwing grenade bandolier

### PR DESCRIPTION
**Pull request by EirenexTheDragon**
A piece of clothing that should resemble what soldiers wear, who store their grenades in special pouches, instead of storing it in their inventory.

### CODE REVIEW

File: data/json/recipes/armor/storage.json [hunk 1]
```diff
@@ -797,5 +797,18 @@
...
       [ [ "leather_pouch", 5 ] ]
     ]
+  },
+  {
+    "result": "grenadebandolier",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_STORAGE",
+    "skill_used": "tailor",
+	"skills_required": [ [ "gun", 1 ], [ "throw", 1 ] ],
```
Comment (mlangsdorf) [inline data/json/recipes/armor/storage.json]: Is this line formated right?  It looks like it's insufficiently indented.

File: data/json/items/grenades.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
     "type": "GENERIC",
     "name": "tear gas grenade",
     "price": 3600,
+	"flags": [ "GRENADE" ],
```
Comment (mlangsdorf) [inline data/json/items/grenades.json]: This line isn't indented correctly, but the "GRENADE" flag should be added to the flags list in grenade_canister and shouldn't be here.

File: data/json/items/grenades.json [hunk 3]
```diff
@@ -97,6 +98,7 @@
     "name": "smoke bomb",
     "description": "This is a canister grenade filled with a variety of pyrotechnic chemicals.  Use this item to pull the pin and light the fuse, turning it into an active smoke bomb.  Five turns after you do that, it will begin to expel a thick black smoke.  This smoke will slow those who enter it, as well as obscuring vision and scent.",
     "price": 1800,
+	"flags": [ "GRENADE" ],
```
Comment (mlangsdorf) [inline data/json/items/grenades.json]: Same issue as above.

File: data/json/items/armor/bandolier.json [hunk 4]
```diff
@@ -138,5 +138,32 @@
...
+  },
+  {
+    "id": "grenadebandolier",
+    "type": "ARMOR",
+    "name": "large grenade pouch",
+    "name_plural": "large grenade pouches",
+    "description": "A pouch for holding up to four full-sized grenades of various types.",
+    "weight": 600,
+    "volume": 5,
+    "price": 10000,
```
Comment (Night-Pryanik) [inline data/json/items/armor/bandolier.json]: 10000 for a simple grenade pouch? More than a EMP grenade (6000), or even the ordinary grenade (1500)?

**Comment by EirenexTheDragon** [comment]
So about the Issue with the storing of the Smoke Grenade (active) and Teargas (active) 

the problem is, that using anything in my knowledge did not make it possible to remove the flag upon activitation of these certain grenades. a other issue is, that you are also able to pull the pin with "%" in the holster, which applies to all grenades in that case sadly. the armed state of these grenade are not storeable however to make things clear.

the space fixing and adding scrambler is done, however, i am unable to find a solution for this bug to remove.

another thing is that these active grenade are frozen in time when in the holster for which i know no solution either, for how to make the counter just continue by itself. 

Here the picture of letting you pull the pin of any kind of grenade in the holster
grafik

i am not sure how i can solve the prompt for leaving the option of storing or drawing a grenade just yet, but it seems possible to do with some time.

**Comment by mlangsdorf** [comment]
And I was wrong and that's not the failure: holster doesn't check flags during removal afaict. Will keep digging.

**Comment by EirenexTheDragon** [comment]
The problem with not being able to escape prompt when activating the grenade pouch is apparently already a problem with an other existing item. you can fill a tool belt with various tool until all slots  and activate the belt, it wont let you promt out either.

**Comment by kevingranade** [comment]
It should be quite doable to prohibit stashing active grenades...

**Pull request merged**